config/default.yml

```diff
@@ -482,6 +482,14 @@ Rails/RenderInline:
   Enabled: 'pending'
   VersionAdded: '2.7'
 
+Rails/RenderPlainText:
+  Description: 'Prefer `render plain:` over `render text:`.'
+  StyleGuide: 'https://rails.rubystyle.guide/#plain-text-rendering'
+  Enabled: 'pending'
+  VersionAdded: '2.7'
+  # Convert only when `content_type` is explicitly set to `text/plain`.
+  ContentTypeCompatibility: true
```

Comment:
It does not match the `MimeTypeCompatibility` in the documentation.

Reply:
Ouch